res/CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ add_resources(
     fonts/private/6-stipple-dash.png
     fonts/private/7-stipple-zigzag.png
     fonts/unicode.lff.gz
+    fonts/BitstreamVeraSans-Roman-builtin.ttf
```

Comment:
What's the license for Bitstream Vera? Can you please rename `COPYING.txt` to `COPYING-GPL3.txt` and add a new `COPYING.txt` describing which parts are covered by which license? Ideally that should cover `extlib/` but you can just cover the newly added font and I'll fill in the rest.

Reply:
I'm not sure I understand what you mean. The Bistream Copyright notice should be included in the distribution of SolveSpace; adding it as a resource (i.e. inlining it into the executable's data section) doesn't really seem like the best idea here.
Or do you mean in the project root? Though then I would name the file 'THIRD_PARTIES.txt' and list all third party licenses/copyrights etc there.